examples/rust/minimal_serve/README.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Minimal `serve` example
+python: https://github.com/rerun-io/rerun/tree/latest/examples/python/minimal_serve/main.py
```

Comment:
that doesn't exist, did you mean to add it to this PR?

Reply:
Nope, this is a Rust only thing. Removed the python line.